catchpoint/dashboards/Catchpoint-Overview.json

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
I noticed there's a few spots in this file where a particular account id is referenced. Since this can be installed by any account, we need to set these to `0`. There's a handy little helper script you can run to clean up dashboard json files, instructions [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboard-json)